Celeste.Mod.mm/Mod/UI/OuiModToggler.cs

```diff
@@ -436,7 +436,29 @@ class OuiModToggler : OuiGenericMenu, OuiModOptions.ISubmenu {
                         }
                     }
 
-                    // restart the game
+                    // does the modified blacklist contain strictly un-blacklisted mods?
+                    if (Everest.Flags.SupportRuntimeMods && blacklistedMods.IsSubsetOf(blacklistedModsOriginal)) {
+                        foreach (string mod in blacklistedModsOriginal.Except(blacklistedMods)) {
+                            try {
+                                Logger.Log("OuiModToggler", $"Attempting to load {mod} after enabling");
```

Comment:
LogLevels should be explicitly set

Reply:
162574b6c739f5696b21fbda5606671fa7d10d61